api/users/views.py

```diff
@@ -436,3 +437,66 @@ def perform_destroy(self, instance):
             if val['id'] in current_institutions:
                 user.remove_institution(val['id'])
         user.save()
+
+
+class UserIdentitiesList(JSONAPIBaseView, generics.RetrieveAPIView, UserMixin):
+    """
+    REMEMBER TO WRITE DEV DOCS!!!
+    The documentation for this endpoint can be found [here](https://developer.osf.io/#operation/...).
+    """
+    permission_classes = (
+        base_permissions.TokenHasScope,
+        drf_permissions.IsAuthenticatedOrReadOnly,
+        CurrentUser,
+    )
+
+    serializer_class = UserIdentitiesSerializer
+
+    required_read_scopes = [CoreScopes.USERS_READ]
```

Comment:
Wondering if we should define more specific scopes for user settings, i.e. CoreScopes.USER_SETTINGS_READ, CoreScopes.USER_SETTINGS_WRITE.

Reply:
Agreed. I'll plan to merge #8527 first so the CoreScopes added in that PR can be used here.